src/main/java/duke/Parser.java

```diff
@@ -16,6 +16,9 @@ public static Command parse(String fullCommand) throws EmptyDescriptionException
             case ("done"):
                 int index = Integer.parseInt(fullCommand.split(" ")[1]) - 1;
                 return new DoneCommand(index);
+            case("find"):
+                String keyword = strArr[1].trim();
```

Comment:
For a possible additional feature, allow for multi-word searching. This can only search for a single word/letter. One method would be to concatenate the remainder of the array elements in your `strArr`.